cmd/mailchain/internal/settings/testdata/TestRoot_ToYaml/comment-defaults.comment-defaults.golden.yaml

```diff
@@ -122,6 +122,12 @@
 #     address: "https://relay.mailchain.xyz/json-rpc/ethereum/rinkeby"
 #   ethereum-rpc2-ropsten:
 #     address: "https://relay.mailchain.xyz/json-rpc/ethereum/ropsten"
+#   substrate-rpc-edgeware-testnet:
```

Comment:
It looks like `substrate-rpc-edgeware-testnet` should now be called `substrate-rpc-edgeware-berlin` see https://polkadot.js.org/apps/#/explorer with an address of `wss://berlin1.edgewa.re`